app/src/test/java/com/codesoom/assignment/domain/UserTest.java

```diff
@@ -0,0 +1,36 @@
+package com.codesoom.assignment.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class UserTest {
+
+    private static final Long ID = 1L;
+    private static final String NAME = "홍길동";
+    private static final String EMAIL = "email";
+    private static final String PASSWORD = "password";
+
+    @DisplayName("기본 생성자는 회원 인스턴스를 만들 수 있다.")
```

Comment:
"만들 수 있다"보다 더 강한 표현을 사용해 봅시다.
```suggestion
    @DisplayName("기본 생성자는 회원 인스턴스를 리턴한다.")
```